status_codes.go

```diff
@@ -0,0 +1,13 @@
+package reggie
```

Comment:
For this file, lets rename it to `http_methods.go`. Also, please run gofmt so the consts line up nice